Updated "Speeding up Vault Operations" section (#57701)
ansible/ansible: docs/docsite/rst/user_guide/vault.rst

Speeding Up Vault Operations
````````````````````````````

> If there are many encrypted files, decrypting them at startup may cause a perceptible delay. To speed this up, cryptography package needs to be installed:

Reviewer: ```suggestion
If you have many encrypted files, decrypting them at startup may cause a perceptible delay. To speed this up, install the cryptography package:
```
